src/extension.ts

```diff
@@ -9,11 +9,12 @@ async function pickConfig(manager: Manager, activeOrNot?: boolean) {
   if (activeOrNot === false) {
     names = others.filter(c => !names.find(cc => cc.name === c.name));
   } else if (activeOrNot === undefined) {
-    others.forEach(n => names.indexOf(n) === -1 && names.push(n));
+    names = others;
```

Comment:
This change would make it that *all saved* configs are shown, but in case of deleted but still active configs, they wouldn't be listed. This would make e.g. the "Disconnect" command possibly not list them.

This is unrelated to #94 as far as I can tell. The command missing has something to do with the extension not being started when the command is invoked from the Command Palette. This is unrelated to how the list of configurations is created.

As for issue #95, I took a look and implemented a fix (db576ac) under version 1.12.0 of the extension.